policyengine_api/endpoints/economy/economy.py

```diff
@@ -50,9 +51,6 @@ def get_economic_impact(
         dict: The economic impact.
     """
     print(f"Got request for {country_id} {policy_id} {baseline_policy_id}")
```

Comment:
With this change, this line will no longer print for invalid countries. Is it important to preserve this behavior (ie: we want to print all requested countries)? If so, perhaps it's best to add this printing/logging elsewhere. Otherwise, perhaps it can be removed entirely?

Reply:
No, I don't think it's important to maintain this logging. Removing is totally fine.

Reply:
It looks like this function moved to `policyengine_api/routes/economy_routes.py`. I've made the update there and removed his comment.